front-end/src/components/VPLink/VPLinkWidget.js

```diff
@@ -1,5 +0,0 @@
-import { DefaultLinkWidget } from '@projectstorm/react-diagrams';
```

Comment:
Same comment as above. It seems fine to delete, but just wanted to double-check this doesn't break anything.